### DISCUSSION TITLE
Using html links and upload feature

### DISCUSSION THREAD

**Discussion by blndfrncsc** [globaleaks/globaleaks-whistleblowing-software#3800] [Q&A]
Hi All,
we're having two issues with our Globalleaks instance (version 4.2.12):
- using the attachment question type in the same step 2 times doesen't seem working: when you upload a file on the second occurrence it loads the file in the first attachment field
-  using html links in questions don't work although  links do work in  site settings presentation; using showdown markdown doesn't seem to work either
Thank for helping.
Francesco

**Comment by evilaliv3** [comment]
Hello.

Version 4.2.12 is quite outdated now. If it possible for you to update to the latest release?
